topics/galaxy-interface/tutorials/workflow-parameters/tutorial.md

```diff
@@ -294,7 +293,7 @@ In the following example we will construct a straightforward workflow, and utili
 
 > <hands-on-title>Running the main workflow</hands-on-title>
 >
-> 1. Upload/reuse the tabular file from the following [section](#running-the-workflow-2)
+> 1. Upload/reuse the tabular file from the following [section](#hands-on-running-the-workflow-1)
```

Comment:
```suggestion
> 1. Upload/reuse the tabular file from the following [section](#hands-on-running-the-workflow)
```

your `-1` suggests to me you're running the training materials locally, in `--incremental` mode. (or `make serve-quick`) there is a known issue whereby if you're running it locally, it will append incrementing numbers to the end of IDs. Those numbers should only appear for 'duplicate' header IDs, but due to how the incremental mode works, every time that page is rebuilt, the number is increased. The online version of the material uses `hands-on-running-the-workflow` without the `-1`, and given that you haven't added another section with that ID, that should be the correct final value. Sorry about this odd bug!

Reply:
Yes i noticed that difference between local and online.
Actually i am referring to this section here: https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/workflow-parameters/tutorial.html#hands-on-running-the-workflow-1
It is the second 'Running the workflow' header.
Thank for reviewing :)